documentation/src/main/asciidoc/topics/proc_using_jboss_marshaller.adoc

```diff
@@ -1,17 +1,10 @@
-[id='jboss_marshalling']
+[id='using-jboss-marshaller_{context}']
 = Using JBoss Marshalling
-JBoss Marshalling is a serialization-based marshalling library and was the default marshaller in previous {brandname} versions.
+JBoss Marshalling is a serialization-based marshalling library and was the default marshaller in previous {brandname} versions but is now deprecated.
 
 [NOTE]
 ====
-* You should not use serialization-based marshalling with {brandname}. Instead
-you should use Protostream, which is a high-performance binary wire format that
-ensures backwards compatibility.
-
-* JBoss Marshalling and the `AdvancedExternalizer` interface are deprecated and
-will be removed in a future release. However, {brandname} ignores
-`AdvancedExternalizer` implementations when persisting data unless you use
-JBoss Marshalling.
+Use JBoss Marshalling only if your use case requires the `AdvancedExternalizer` interface, which is now deprecated but depends on JBoss Marshalling, or if you are temporarily using JBoss Marshalling to migrate your client applications from an older version of {brandname}.
```

Comment:
Can we go further and explicitly say that jboss-marshalling itself is deprecated.